lumibot/brokers/broker.py

```diff
@@ -723,8 +724,8 @@ def _process_trade_event(
                     filled_quantity = Decimal(filled_quantity)
                 if filled_quantity < 0:
                     raise error
-            except:
-                raise error
+            except ValueError:
+                raise error from None
```

Comment:
If you are going to bother with the "from" notation, I'd actually include the original info:
```python
except ValueError as e:
    raise error from e
```
alternatively, just leaving it off is fine as well
```python
except ValueError:
    raise error
```